CONTRIBUTING.md

```diff
@@ -68,18 +68,18 @@ and it will be automatically reflected in the PR.
 
 
 ### Environment Setup
-Flame uses [Melos] to manage the project and dependencies.
 
-To install Melos, run the following command from your terminal:
+Flame uses [Melos] to manage the project and dependencies. To install Melos, run the following
+command from your terminal:
 
 ```bash
-flutter pub global activate melos
+$ flutter pub global activate melos
```

Comment:
I find that adding these makes it harder for users to copy the line, and I think people understand that they shouldn't run it as root. (And if they don't understand that they should run it in a shell they probably don't know what `$` is anyways 😅)

Reply:
Right, I'll put them back for the CONTRIBUTING doc. In other docs the `$` sign is not selectable, so it doesn't interfere with anything.